hack/generator/pkg/astmodel/object_type.go

```diff
@@ -378,6 +378,15 @@ func (objectType *ObjectType) WithFunction(function Function) *ObjectType {
 	return result
 }
 
+// WithoutFunction creates a new ObjectType without the specified function
+func (objectType *ObjectType) WithoutFunction(name string) *ObjectType {
```

Comment:
I saw this earlier and wasn't sure why we need it - sorta feels as though this is a way to paper over a deeper issue.

Reply:
Removed this in favor of a better way